docs/source/develop.rst

```diff
@@ -366,6 +366,7 @@ Distributed submit PRs.  In this case, the gpuCI bot will comment on the PR:
 .. note:: Can one of the admins verify this patch?
 
 .. image:: images/gputester-msg.png
+   :alt: ""
```

Comment:
```suggestion
   :alt: "Screenshot of a GitHub comment left by the GPUtestter bot, where the comment says 'Can one of the admins verify this patch?'."
```

I don't think this alt text is supposed to be blank! :laughing: How about this suggestion?

Reply:
Yeah I wasn't sure about this one! At first I thought it was functional, but then according to [these guidelines](https://www.w3.org/WAI/tutorials/images/decorative/) I thought it might be a decorative image, and then changed it to an empty string. Maybe @isabela-pf can weigh in?

Reply:
erring on the side of more info! :)